pages/blog/openforce-2022.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "AsyncAPI at OpenForce"
+date: 2022-02-28T06:00:00+01:00
+type: Communication
+tags:
+  - Open Source
+  - Contribute
+cover: /img/posts/openforce-2022/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Dev Comm Keeper
+featured: true
+---
+
+Two years ago AsyncAPI Initiative actively participated in Hacktoberfest to help folks start their journey in open source. This was a very successful event. We got feedback that the level of commitment that we presented makes us perfect candidates for Google Summer of Code. 
+
+Last year we participated in Google Summer of Code and again in Hacktoberfest. It went great! We onboarded many new super enthusiastic contributors that stayed with us longer. 
+
+This year could not be different. We want to do it again and yet again on a larger scale. Thus active participation in initiatives focused on introducing new folks in open source became [one of our main goals for 2022](https://github.com/asyncapi/community/discussions/193).
+
+Thanks to one of our contributors, [Hargun Kaur](https://www.linkedin.com/in/hkaur008/) we learned about [OpenForce](https://www.openforce.tech/) and decided to join it in March 2022.
+
+## What we offer
+
+AsyncAPI maintainers prepared some GitHub issues that are easy enough to be completed in one month. We offer full support in discussing possible solutions and on-time review in pull requests.
+
+AsyncAPI community is a safe place for your first open-source contribution. Many folks are open to support your first steps in this great world.
+
+In exchange, every person that completes these tasks will get an AsyncAPI t-shirt!
+
+## How to connect
+
+Join [our Slack workspace](https://www.asyncapi.com/slack-invite). Just make sure to follow our [Slack etiquette](https://github.com/asyncapi/.github/blob/master/slack-etiquette.md) and [the code of conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md).
+
+We will also have [Abir Pal](https://twitter.com/imabptweets) present in OpenForce Discord channel, in case you need help on your way to AsyncAPI.
```

Comment:
```suggestion
We will also have [Abir Pal](https://twitter.com/imabptweets) present in the OpenForce Discord channel, in case you need help on your way to AsyncAPI.
```